spring-cloud/pom.xml

```diff
@@ -39,6 +39,7 @@
         <module>spring-cloud-task</module>
         <module>spring-cloud-zuul</module>
         <module>spring-cloud-zuul-fallback</module>
+        <module>spring-cloud-open-service-broker</module>
```

Comment:
Let's move this entry so that it's in alphabetical order among the other modules

Reply:
I've moved it under spring-cloud-kubernetes, but I've seen that the modules are not quite in alphabetical order:
`<module>spring-cloud-config</module>
 <module>spring-cloud-eureka</module>
 <module>spring-cloud-hystrix</module>
 <module>spring-cloud-bootstrap</module>
 <module>spring-cloud-ribbon-client</module>`